container/service.proto

```diff
@@ -10,46 +10,51 @@ import "container/types.proto";
 import "refs/types.proto";
 import "session/types.proto";
 
-// ContainerService provides API to access container smart-contract in morph chain
-// via NeoFS node.
+// `ContainerService` provides API to interact with `Container` smart contract
+// in NeoFS sidechain via other NeoFS nodes. All of those actions can be done
+// equivalently by directly issuing transactions and RPC calls to sidechain
+// nodes.
 service ContainerService {
-  // Put invokes 'Put' method in container smart-contract and returns
-  // response immediately. After new block in morph chain, request is verified
-  // by inner ring nodes. After one more block in morph chain, container
-  // added into smart-contract storage.
+  // `Put` invokes `Container` smart contract's `Put` method and returns
+  // response immediately. After a new block is issued in sidechain, request is
+  // verified by Inner Ring nodes. After one more block in sidechain, container
+  // is added into smart contract storage.
```

Comment:
I am not sure but maybe it is worth to mention, that smart contract transfers internal assets to the inner ring nods therefore it is payed operation.